src/Stratis.Bitcoin.Networks/StraxMain.cs

```diff
@@ -33,9 +33,6 @@ public StraxMain()
             this.CoinTicker = "STRAX";
             this.DefaultBanTimeSeconds = 11250; // 500 (MaxReorg) * 45 (TargetSpacing) / 2 = 3 hours, 7 minutes and 30 seconds
 
-            // TODO: Update this later
```

Comment:
I think we should still retain some sort of TODO here; the mainnet federation will need to be defined

Reply:
Added the federation to the mainchain as well.